### PR TITLE
feat(Resource-status/Listing): Implement action URL & notes label

### PR DESCRIPTION
**Pull request by bdauria**
## Description

This adds a column for the action URL and hooks up the Notes label when the notes URL is hovered:

image

**Fixes** # (issue)

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 21.04.x (master)

### CODE REVIEW

**Comment by adr-mo** [comment]
Looks good to me for the backend 🙂

File: www/front_src/src/Resources/Listing/columns/Url/Action.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import UrlColumn from '.';
+
+const ActionUrlColumn = ({ row }: ComponentColumnProps): JSX.Element => {
+  const endpoint = path<string | undefined>(
+    ['links', 'externals', 'action_url'],
+    row,
+  );
+
+  const title = 'action';
```
Comment (Thebarda) [inline www/front_src/src/Resources/Listing/columns/Url/Action.tsx:17]: This does not print the action URL

**Review by Thebarda — COMMENTED** [review]
I'v tested it and is it possible to open action & note link in another tab ?

**Comment by bdauria** [comment]
> I'v tested it and is it possible to open action & note link in another tab ?

Yes, by using the third mouse button

File: www/front_src/src/Resources/Listing/columns/Url/Action.tsx (hunk 1 above)
Comment (bdauria) [reply www/front_src/src/Resources/Listing/columns/Url/Action.tsx]: Fixed

**Review by Thebarda — DISMISSED** [review]

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -16026,3 +16026,6 @@ msgstr "Commande de contrôle de l'id de service %d non trouvée"
 
 msgid "Error while searching for the command of host"
 msgstr "Erreur lors de la recheche de la commande de l'hôte"
+
+msgid "Monitoring server"
+msgstr "Server de supervision"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:16031]:
```suggestion
msgstr "Serveur de supervision"
```

File: www/front_src/src/Resources/Listing/columns/Url/Notes.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+import UrlColumn from '.';
+
+const NotesUrlColumn = ({ row }: ComponentColumnProps): JSX.Element => {
+  const endpoint = path<string | undefined>(
+    ['links', 'externals', 'notes', 'url'],
+    row,
+  );
+
+  const title = path<string | undefined>(
```
Comment (kduret) [inline www/front_src/src/Resources/Listing/columns/Url/Notes.tsx:17]: Title should be url if label is not set

**Review by kduret — APPROVED** [review]

**Review by Jordan91 — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**